llvm/test/CodeGen/AArch64/GlobalISel/combine-commute-int-const-lhs.mir

```diff
@@ -0,0 +1,28 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
```

Comment:
Can you merge this test with test/CodeGen/AArch64/GlobalISel/combine-const-fold-barrier-rhs.mir

Reply:
> Can you merge this test with test/CodeGen/AArch64/GlobalISel/combine-const-fold-barrier-rhs.mir

This test needs `--aarch64prelegalizercombiner-disable-rule=constant_fold_binop` to trigger issue. That makes the merge complicated. Shall we leave as it is.